### PR TITLE
[interp] Implement tracing of return values. Improve printing of arrays.

### PR DESCRIPTION
**Pull request by vargaz**

### CODE REVIEW

**Comment by vargaz** [comment]
@monojenkins build failed

File: mono/mini/ee.h [hunk 1]
```diff
@@ -15,7 +15,7 @@
 #ifndef __MONO_EE_H__
 #define __MONO_EE_H__
 
-#define MONO_EE_API_VERSION 0xa
+#define MONO_EE_API_VERSION 0xc
```
Comment (lewurm) [inline mono/mini/ee.h]: `0xa + 0x1 = 0xb`? 😅

**Comment by vargaz** [comment]
@monojenkins build failee

**Comment by vargaz** [comment]
@monojenkins build failed

**Comment by vargaz** [comment]
@monojenkins build failed

**Pull request merged**